pkg/bufferdecoder/protocol.go

```diff
@@ -23,29 +23,29 @@ const (
 // NOTE: Integers want to be aligned in memory, so if changing the format of this struct
 // keep the 1-byte 'Argnum' as the final parameter before the padding (if padding is needed).
 type Context struct {
-	Ts              uint64
-	StartTime       uint64
-	CgroupID        uint64
-	Pid             uint32
-	Tid             uint32
-	Ppid            uint32
-	HostPid         uint32
-	HostTid         uint32
-	HostPpid        uint32
-	Uid             uint32
-	MntID           uint32
-	PidID           uint32
-	Comm            [16]byte
-	UtsName         [16]byte
-	Flags           uint32
-	EventID         events.ID // int32
-	Syscall         int32
-	MatchedPolicies uint64
-	Retval          int64
-	StackID         uint32
-	ProcessorId     uint16
-	Argnum          uint8
-	_               [1]byte // padding
+	Ts                    uint64
+	StartTime             uint64
+	CgroupID              uint64
+	Pid                   uint32
+	Tid                   uint32
+	Ppid                  uint32
+	HostPid               uint32
+	HostTid               uint32
+	HostPpid              uint32
+	Uid                   uint32
+	MntID                 uint32
+	PidID                 uint32
+	Comm                  [16]byte
+	UtsName               [16]byte
+	Flags                 uint32
+	EventID               events.ID // int32
+	Syscall               int32
+	MatchedPoliciesKernel uint64
```

Comment:
I think for the protocol this can still be called matchedPolicies without kernel